website/content/en/docs/upgrading-sdk-version/v1.36.0.md

```diff
@@ -5,121 +5,124 @@ weight: 998964000
 
 ## Upgrade K8s versions to use 1.29
 
-Regarding the upcoming major changes in the plugin, please note that `go/v3` is being deprecated by Kubebuilder hence Operator SDK would also be migrating to `go/v4` soon. Following are the changes to be addressed in your project structure, `Makefile` and `go.mod` files: 1) [go/v4] Update your `go.mod` file to upgrade the dependencies and run `go mod tidy` to download them ```go
-  k8s.io/api v0.29.2
-  k8s.io/apimachinery v0.29.2
-  k8s.io/client-go v0.29.2
-  sigs.k8s.io/controller-runtime v0.17.3
-``` 2) [go/v4, helm/v1] The projects are now scaffolded with 0.16.0 version of kube-rbac-proxy. Modify the version of kube-rbac-proxy in the following scaffolded files: - config/default/manager_auth_proxy_patch.yaml     
-  ```diff
-    - gcr.io/kubebuilder/kube-rbac-proxy:v0.15.0
-    + gcr.io/kubebuilder/kube-rbac-proxy:v0.16.0
-  ```
+Regarding the upcoming major changes in the plugin, please note that `go/v3` is being deprecated by Kubebuilder hence Operator SDK would also be migrating to `go/v4` soon. Following are the changes to be addressed in your project structure, `Makefile` and `go.mod` files:
+1) [go/v4] Update your `go.mod` file to upgrade the dependencies and run `go mod tidy` to download them:
+   ```go
+   k8s.io/api v0.29.2
+   k8s.io/apimachinery v0.29.2
+   k8s.io/client-go v0.29.2
+   sigs.k8s.io/controller-runtime v0.17.3
+   ```
+2) [go/v4, helm/v1] The projects are now scaffolded with 0.16.0 version of kube-rbac-proxy. Modify the version of kube-rbac-proxy in the following scaffolded files: - config/default/manager_auth_proxy_patch.yaml     
+   ```diff
+     - gcr.io/kubebuilder/kube-rbac-proxy:v0.15.0
```

Comment:
I would remove spaces from before `-` or  `+` in diff code blocks, so they render with syntax highlighting

Example without spaces (from https://deploy-preview-6797--operator-sdk.netlify.app/docs/upgrading-sdk-version/v1.36.0/)
![Screenshot from 2024-08-03 21-37-48](https://github.com/user-attachments/assets/313a9df4-1e99-445e-8b4f-7ce23d65ba78)

Reply:
Good call, I'll do that in this pr too.